docs/framework/configure-apps/file-schema/winforms/windows-forms-add-configuration-element.md

```diff
@@ -33,33 +33,33 @@ The following sections describe attributes, child elements, and parent elements.
 
 ### Attributes
```

Comment:
i've noticed this topic and some others are missing the ms.prod value which is a required metadata. we should file an issue to take care of that.